code/modules/vending/wardrobes.dm

```diff
@@ -87,6 +87,7 @@
 					/obj/item/storage/backpack/satchel/eng = 3,
 					/obj/item/clothing/suit/hooded/wintercoat/engineering = 3,
 					/obj/item/clothing/suit/hooded/wintercoat/engineering/tcomms = 3,
+					/obj/item/clothing/suit/hooded/wintercoat/engineering/void = 3,
```

Comment:
goddamit kirie we have a file to add things to vending machines

Reply:
With this it's kinda meh, keeps it all in one place

Reply:
The other outfits though, gets really messy all in one file